src/python/lib/ensembl/compara/runnable/SymlinkFasta.py

```diff
@@ -0,0 +1,67 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runnable to symlink a fasta file into a set directory.
+
+This runnable symlinks a fasta file into a designated directory.
+
+Optionally cleans up broken symlinks first. By default cleanup_symlinks is False.
+
+"""
+
+import subprocess
+
+from typing import Dict
+
+import eHive
+
+class SymlinkFasta(eHive.BaseRunnable):
+    """Symlinks a fasta file into a specified directory"""
+
+    def param_defaults(self) -> Dict[str, bool]:
+        """set default parameters"""
+        return {
+            'cleanup_symlinks' : False,
```

Comment:
```suggestion
            'cleanup_symlinks': False,
```

Reply:
This is to make the code PEP8 compliant.

Reply:
Comments with reasons for formatting suggestions would have helped filter the original review suggestions. Or - perhaps for the future, only formatting suggestions with valid reasons should be included in the review in the first place.